forage-android/src/main/java/com/joinforage/forage/android/core/EnvConfig.kt

```diff
@@ -0,0 +1,98 @@
+package com.joinforage.forage.android.core
+
+import com.joinforage.forage.android.ui.ForageContext
+
+enum class EnvOption {
+    DEV,
+    STAGING,
+    SANDBOX,
+    CERT,
+    PROD
+}
+
+sealed class EnvConfig(
```

Comment:
should this be `internal`? Will the btProxyId, be accessible by the client?

Reply:
yessir! Good catch!

Reply:
address in [eb1c88b](https://github.com/teamforage/forage-android-sdk/pull/102/commits/eb1c88b6dd6a90abca84fbb47959e992b3878673)

Reply:
somehow I only made the change for STAGINGing and not SANDBOXbox. Will update this per with the typo fix for SANDBOXbox as well